packages/eventsub-base/src/events/EventSubStreamOnlineEvent.ts

```diff
@@ -65,6 +65,13 @@ export class EventSubStreamOnlineEvent extends DataObject<EventSubStreamOnlineEv
 		return (await this._client.streams.getStreamByUserId(this[rawDataSymbol].broadcaster_user_id))!;
 	}
 
+	/**
+	 * The ID of the stream going live.
+	 */
+	get streamId(): string {
```

Comment:
Should just be named `id` IMO. The event already refers to a stream, so no need to qualify.

Reply:
Well, I can rename it. But I chose this name to be consistent with `streamType` getter.

```ts
get streamType(): EventSubStreamOnlineEventStreamType {
    return this[rawDataSymbol].type;
}
```

Is it ok?

Reply:
Fair enough, I will probably have to clean up a bunch of naming inconsistencies when I can. As for this one, please just rename it to `id`.

Reply:
Done 👀